classes/Route.php

```diff
@@ -4,6 +4,7 @@ class Route {
     const NOT_IN_VILLAGE = 0;
     const IN_VILLAGE_OKAY = 1;
     const ONLY_IN_VILLAGE = 2;
+    const VILLAGE_OR_ARENA = 3;
```

Comment:
```suggestion
    const VILLAGE_OR_COLOSSEUM = 3;
```
Not to be confused with the normal arena